website_cookiebot/models/website.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Trey - Antonio González <antonio@trey.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    cookiebot_id = fields.Char(
```

Comment:
This is not a requirement, but would you be able to name the field differently?

Fields ended with `_id` are usually Many2one fields. To avoid confusion, you could call this field `cookiebot_identifier` for example.